### PR TITLE
Post mtg cleanup

### PR DESCRIPTION
**Pull request by lindsaycarr**
Basically re-doing the entire viz.yaml and structure after we met on 10/19. 

This PR has some viz ids commented out b/c they are trying to pull down geotiff from GDP and are failing. By commenting those pieces out, this PR will run all the way through `make` and have a working target. These commented out sections should be fixed with #35.

This will be the new starting place and I will update/close issues based on content in this PR.

### CODE REVIEW

File: scripts/visualize/visualize.map_geometry.R [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  
+  deps <- readDepends(viz)
+  geom_feature <- deps[['geom_feature']]
+  geom_context <- deps[['geom_context']]
+  
+  library(sp)
+  
+  png(viz[['location']])
+  plot(geom_context, col="grey")
+  plot(geom_feature, col="lightyellow")
```
Comment (wdwatkins) [inline scripts/visualize/visualize.map_geometry.R]: don't need an `add=TRUE` here?

File: scripts/fetch/fetch.geom_transpose.R [hunk 2]
```diff
@@ -0,0 +1,9 @@
+fetch.geom_transpose <- function(viz){
+  deps <- readDepends(viz)
+  
+  sp_poly_transpose <- raster::shift(deps[["geom_sp"]], y = viz[["vertical_trans_va"]])
```
[the remark this answers is not in the document]
Comment (lindsaycarr) [reply scripts/fetch/fetch.geom_transpose.R]: The raster function was the only one I could find when I searched. I can use `maptools` instead!

File: scripts/visualize/visualize.map_geometry.R (hunk 1 above)
Comment (lindsaycarr) [reply scripts/visualize/visualize.map_geometry.R]: oh yep

Comment (lindsaycarr) [reply scripts/visualize/visualize.map_geometry.R]: these aren't totally perfect - need to work on limits and stuff. Just wanted something up and running.

Comment (wdwatkins) [reply scripts/visualize/visualize.map_geometry.R]: 👍  go ahead an merge if you are ready

**Pull request merged**